example-config.yml

```diff
@@ -15,6 +15,11 @@ api:
 #     lon: -122.71607145667079
 #     name: Oregon Zoo, Portland, OR
 
+### The default query parameters can be overridden be uncommenting this object.
+### Note: the override values must be valid values within otp-ui's query-params.js
+# defaultQueryParams:
+#   maxWalkDistance: 3219 # 2 miles in meters
```

Comment:
The UI does not seem to be set on the default query params, does this PR intend to do that?

As an example, if you set `optimize: TRANSFERS`, the optimization setting will still appear as `SPEED` in the UI (with a red dot indicating that that is not the default setting, though.)